source/downloads.html

```diff
@@ -118,6 +118,18 @@ <h2 id="prior-releases">Prior Releases</h2>
       <a href="{{ site.wiki_url }}/Version+Notes+2.5.13">Version notes</a>
     </td>
   </tr>
+  <tr>
+    <td class="no-wrap">
+      Struts 2.3.34
+    </td>
+    <td class="no-wrap">September 2017</td>
+    <td>
+    </td>
+    <td>
+      <a href="/docs/version-notes-2334.html">Version notes</a>
```

Comment:
Could you use `{{ site.wiki_url }}/Version+Notes+2.3.34`? And yes, according to our records 2.3.34 is out of vulnerabilities

Reply:
I just copied from previous.
Switching to `{{ site.wiki_url }}` for all should be other PR

Reply:
Ok but please bear in mind that :)